core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7143,6 +7161,10 @@ public static final class Name {
         "alluxio.master.metadata.sync.instrument.executor";
     public static final String MASTER_METADATA_SYNC_REPORT_FAILURE =
         "alluxio.master.metadata.sync.report.failure";
+    public static final String MASTER_METADATA_SYNC_TRAVERSE_TYPE =
+        "alluxio.master.metadata.sync.traverse.type";
+    public static final String MASTER_METADATA_SYNC_PREFETCH_CHILDREN_UFS_STATUS =
```

Comment:
are there use cases you need to turn off prefetch?

Reply:
A directory called A has 10,000 subdirectories, each with 10,000 files. If prefetch is not turned off, then at syncExistingInodeMetadata(A), the ufsStatus of the entire grandchild-level file will be asynchronously obtained.

Without closing prefetch, the meaning of using DFS is weakened.

Reply:
cool tks for the explanation :)